utils/constant.go

```diff
@@ -0,0 +1,3 @@
+package utils
+
+const AuthHeader = "Your-Auth-Header"
```

Comment:
This constant shouldn't exist. I don't think `const` can even be changed by users in Go? When users use this code they're not adding code into this repository. They're importing this library and using the functions we export, so it's not at all easy for them to discover where to set the auth header. You should instead force them to set the auth header in such a way that it's obviously how they can change it –– either by requiring them to include it as a parameter in the function call, or loading it from an environment variable. In that case, you can output a clear error message if no variable is set. There's no way to discover that this wrong until you make a request to serum-api